packages/gestalt/src/Button.js

```diff
@@ -214,6 +216,7 @@ const ButtonWithForwardRef: React$AbstractComponent<
         onClick={handleLinkClick}
         ref={innerRef}
         rel={rel}
+        tabIndex={tabIndex}
```

Comment:
@AlbertCarreras we should also add `tabIndex` to the `<button />` element below

Reply:
DONE